hangman/README.md

```diff
@@ -1,3 +1,5 @@
-8k students only:
 
-Describe how to run the multinode app.
+
+I choose permanent as restart value.
```

Comment:
# -3

I think it should be transient. If a server crashes, you have lost the game state, so there's no point in restarting it.